playbooks/tests/tasks/integration.yml

```diff
@@ -16,29 +16,17 @@
     public_key: "{{ rootuser.ssh_public_key }}"
     state: present
 
-- name: generate test security group
```

Comment:
The issue we are seeing with this task is that ursula gets stuck here. The task doesn't actually fail. Jenkins abort the whole CI job eventually because it takes too long. So I don't think retries here would help.